src/test/ledger/env.ts

```diff
@@ -24,11 +28,16 @@ export interface Enviroment {
 	ethHolder: string;
 	erc20Holder: string;
 	erc721Holder: string;
+	perunArt: string;
 	op: Wallet;
 	tee: Wallet;
 	users: Wallet[];
 }
 
+const PERUNART_NAME = "PerunArt";
+const PERUNART_SYMBOL = "PART";
+const PERUNART_URI = "https://nifty.erdstall.dev";
```

Comment:
always needs trailing `/` because it's concatenated using `abi.encodePacked(uriBase, tokenId)`
```suggestion
const PERUNART_URI = "https://nifty.erdstall.dev/";
```

Reply:
:eagle: :eye: